cypress/objects/taskPage.js

```diff
@@ -0,0 +1,36 @@
+class TaskPage {
+
+    inputIntoASpecificField(fieldId, input) {
+        cy.get(`[id="${fieldId}"]`).type(input);
+    }
+
+    generateRandomBillToClientSelection() {
+        let num = Math.floor(Math.random()*2);
+        if (num === 1)
+            return 'Yes';
+        else
+            return 'No';
+    }
+
+    clickBillToClientDropDown() {
+        cy.get('[class = Select-value]').click();
+    }
+
+    selectRandomBillToClientOption(selection) {
+        cy.get(`[aria-label= "${selection}"]`).click();
+    }
+
+    checkIfBillToClientIsYes() {
+        return cy.get('[class="Select has-value is-searchable Select--single"]').get('[taskDetailsForm.billable]').contains(true);
```

Comment:
not sure what you wanted to achieve with this contains(true), maybe .should('be.visible') would be better?

Reply:
or if you want to see it contains "Yes"